source/extensions/common/wasm/wasm_vm.h

```diff
@@ -41,6 +41,12 @@ using WasmVmPtr = std::unique_ptr<proxy_wasm::WasmVm>;
 // Create a new low-level Wasm VM using runtime of the given type (e.g. "envoy.wasm.runtime.wavm").
 WasmVmPtr createWasmVm(absl::string_view runtime);
 
+// Check if the provided Wasm Engine is compiled with Envoy
```

Comment:
s/Check/@return true/

and use 
```
/**
 * ...
 */
```
for doxygen format.

Reply:
Done